pycloudlib/ec2/instance.py

```diff
@@ -127,65 +123,6 @@ def delete(self, wait=True):
         if wait:
             self.wait_for_delete()
 
-    def restart(self, wait=True, **kwargs):
```

Comment:
Do we know any reason why EC2 had this particular implementation?

Reply:
After reading through this implementation, I don't _think_ we're losing anything by switching to the stop-then-start implementation. But it looks like @paride was the last one to touch this method so he may have a more definitive opinion (though it appears the last significant edit was about 2 years ago)

Reply:
Looking around, the main downside I've found to stop/start is that the IP address will change, but I don't think that will affect us.
The other downside is that you lose anything stored on "ephemeral"/"instance" storage, but you have to go out of your way to use that, and I don't think we're using it at all (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes)